decisions/DR-update-devkit-architecture.md

```diff
@@ -0,0 +1,63 @@
+# Update devkit architecture
+
+| Status      | draft |
+| :---------- | :-------------------------------------------------------------------------------------- |
+| **PR #**    | []() |
+| **Author & Sponsor** | [Marko](https://github.com/m4rk055) (marko@blindnet.io) |
+| **Reviewers** | [Milan](https://github.com/milstan), [Amaury](https://github.com/TheKinrar), [Justin](https://github.com/jboileau99) |
+| **Assignee** | [Milan](https://github.com/milstan) |
+
+## Context and Problem Statement
+
+The model described in the [PRIV](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md) document requires the [Privacy Compiler](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-compiler) to be aware of every [Data Captures and Capture Fragments](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-compiler) in the system.
+That puts a burden on the implementing system to
+- make sure every data point collected by the system is as well referenced (metadata) in the Privacy Computation Engine database
+- collect or transform data to a particular format (capture and fragments)
+
+This DR proposes removing the notion of Data Captures and Capture Fragments from Privacy Computation Engine.
+
+## Proposal
+
+The [Privacy computation engine](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-computation-engine) (PCE) should have the following components:
+- Privacy Compiler
+- Privacy Request Capture Interface
+- Data Consumer/DPO Interface
+- Data access engine
+- Data storage connectors (for e.g. Postgres, MongoDB, Azure Storage)
+
+**Data access engine** is a component deployed by a client. It communicates with the **Privacy Compiler** and a **Data storage connector**.
+
+A **Data storage connector** is a component that has mappings that queries the data storage. It can be implemented by blindnet or by a client for an unsupported storage or if the data model is not suitable.
+It should containg a set of rules (mappings) on how to map a `selector` to a corresponding fetch query (`selector` to a e.g. table column in a relational database). 
```

Comment:
```suggestion
It should containg a set of rules (mappings) on how to map a `selector` to corresponding queries (`selector` to a e.g. table column in a relational database). 
```

We'll have other types of queries than just fetching data, there's some deletions for sure, maybe some updates?